scout/build/disease.py

```diff
@@ -72,12 +36,14 @@ def build_disease_term(disease_info, alias_genes={}):
 
     if hgnc_symbols_not_found:
         LOG.debug(
-            "The following gene symbols could not be found in database: %s", hgnc_symbols_not_found
+            "The following gene symbols could not be found in database: %s",
+            hgnc_symbols_not_found,
         )
 
     disease_obj["genes"] = list(hgnc_ids)
 
-    if "hpo_terms" in disease_info:
-        disease_obj["hpo_terms"] = list(disease_info["hpo_terms"])
+    DiseaseTerm(**disease_obj)
```

Comment:
It's invoking DiseaseTerm (pydantic model class) just for the sake of validating the disease dictionary. It's not returning the pydantic class itself because it would be missing the _id key/value, required when you save the dictionary in the database

Reply:
Well, yes, now it does. 😉